### ISSUE TITLE
Output does not match examples, level not aligned correctly

### ISSUE THREAD

**Issue by evitalis** [Drakulix/simplelog.rs#44]
I noticed today that when using the default configuration output does not seem to match that shown in the examples. The levels with shorter character counts have a leading space which makes it look odd.

Using the slightly modified example from this repo:

```
use log;
use simplelog::*;

fn main() {
    TermLogger::init(LevelFilter::Trace, Config::default(), TerminalMode::Mixed).unwrap();

    log::error!("error");
    log::warn!("warning");
    log::info!("info");
    log::debug!("debug");
    log::trace!("trace");
}
```

I see the following output:

```
$ cargo run
    Finished dev [unoptimized + debuginfo] target(s) in 0.02s
     Running `target/debug/logdemo`
23:29:17 [ERROR] error
23:29:17 [ WARN] warning
23:29:17 [ INFO] info
23:29:17 [DEBUG] (1) logdemo: debug
23:29:17 [TRACE] (1) logdemo: [src/main.rs:11] trace
```

Can the leading space either be removed, or switched to a trailing space to make things look a bit cleaner? I think its there to help the lines look consistent but ends up looking weird at the start.

**Comment by Drakulix** [comment]
Yes, this was introduced by https://github.com/Drakulix/simplelog.rs/commit/084d57a9abb94b1a602ca6917bf834399b77e2b8 .

I can easily make this configurable via the config settings.

**Comment by Drakulix** [comment]
I have introduced a new config option on branch `feature/config_padding` (see #45).
@evitalis can you verify, that this fixes your issue?

If it does, I'll update the examples and release it as 0.7.2 shortly.

**Comment by evitalis** [comment]
Thanks. I can check it out a bit later today. Is it possible to use the defaults and only override this one setting by chance?

**Comment by Drakulix** [comment]
> Thanks. I can check it out a bit later today. Is it possible to use the defaults and only override this one setting by chance?

Yes, just set this in the `ConfigBuilder` and ignore the rest. Then pass the built config to your logger.

**Comment by evitalis** [comment]
Seemed to work for me using my example code as a test. Do you think it may be worth setting the padding to off by default instead of the left padding?

**Comment by Drakulix** [comment]
> Seemed to work for me using my example code as a test. Do you think it may be worth setting the padding to off by default instead of the left padding?

I think I do not want to change the semantics of the default values without a major version break to make people aware. And since `simplelog` had quite a lot major releases recently I will probably delay this decision for now.

Maybe I should open an issue on defaults in general, some other users might have something to say about it as well.

But I am glad this is working, I will release it soon in that case.

**Comment by evitalis** [comment]
Makes sense, maybe on 0.8 the default could be off or right. I am curious why it ends up being left padded though, Western languages usually right pad since they are read left to right.

In any event thanks for the quick update to this.
